packages/core-utils/src/core-utils.story.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { getMostReadableTextColor } from "./route";
+
+export default {
+  title: "core-utils"
+};
+
+const ColorBlock = styled.div<{ bg: string; fg: string }>`
+  background: ${props => props.bg};
+  color: ${props => props.fg};
+  padding: 10px;
+`;
+
+const ColorBlockWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+`;
+
+const ColorPair = ({ fg, bg }: { fg: string; bg: string }) => {
+  return (
+    <ColorBlockWrapper>
+      <ColorBlock fg={fg} bg={bg}>
+        Provided color pair
+      </ColorBlock>
+      <ColorBlock fg={getMostReadableTextColor(bg, fg)} bg={bg}>
+        Corrected color pair
+      </ColorBlock>
+    </ColorBlockWrapper>
+  );
+};
+
+export const RouteColorTester = (): JSX.Element => {
+  const [fg, setFg] = useState("#333333");
```

Comment:
I would be in favor of adding the colors from the unit test too.

Reply:
I just updated my response -- unless I used wild hacks I couldn't get these to save to the stories

Reply:
It doesn't feel like a proper use of stories to me since it's not a component we're testing? Don't stories exist since you can't easily unit test a component? I would think a real unit test is preferred to a story-based solution no?